docs/pages/sdk/snippets/preference-parameters.mdx

```diff
@@ -0,0 +1,63 @@
+
+### preference (optional)
+
+- Type `Preference`
+
+```ts twoslash
+type Attribution = {
+    auto: boolean;
+    dataSuffix?: never;
+} | {
+    auto?: never;
+    dataSuffix: `0x${string}`;
+}
+
+type Preference = {
+    options?: 'all' | 'smartWalletOnly' | 'eoaOnly';
+    attribution?: Attribution;
```

Comment:
we actually just revived `keysUrl` so dev can test against keys-dev, could you also add `keysUrl` here, ty!

Reply:
I wonder if we should update the sdk to guard on this better

Reply:
yea, we are on the fence on this one, on one hand we do know the url is almost permanent, but wonder if we could leave it as a string to keep it flexible for perhaps future new env or url for project-specific closed testing env. 

plus this url param would be dev facing, so we were leaning towards keeping it as string, wdyt

Reply:
Yeah I'll add a note of currently supported values, but leave the type string